classes/class_tilemap.rst

```diff
@@ -608,7 +608,7 @@ Returns ``true`` if the given cell is flipped in the Y axis.
 
 - :ref:`Vector2<class_Vector2>` **map_to_world** **(** :ref:`Vector2<class_Vector2>` map_position, :ref:`bool<class_bool>` ignore_half_ofs=false **)** |const|
 
-Returns the global position corresponding to the given tilemap (grid-based) coordinates.
+Returns the position relative to the origin of the tilemap corresponding to the given tilemap (grid-based) coordinates. To get the global position, use :ref:`to_global<class_Node2D_method_to_global>` on the result of ``map_to_world``.
```

Comment:
i am rather new to godot. would it make sense to put this into an actual example, like `mymap.to_global(mymap.map_to_world(Vector2(3, 3)))` ?